binding/web/src/cheetah.ts

```diff
@@ -487,7 +530,7 @@ export class Cheetah {
     const pv_cheetah_init: pv_cheetah_init_type = this.wrapAsyncFunction(
       module,
       "pv_cheetah_init",
-      5);
+      6);
```

Comment:
we didnt do this, but for cheetah lets add pv_cheetah_delete and wrap it in async as well since it does gatekeeper call on release if it can.